triage/example_database.yaml

```diff
@@ -1,5 +1,5 @@
 host: food_db
-pass: some_password
+pass: your_password
```

Comment:
Thank you!

Hopefully this file will dissapear when `triage` CLI supports reading from the environment